input/resources/NamingSystem-Danish-provider-registry---Ydernummer.json

```diff
@@ -0,0 +1,35 @@
+{
+    "resourceType": "NamingSystem",
+    "id": "ydernummer",
+    "name": "Ydernummer",
+    "status": "active",
+    "kind": "identifier",
+    "date": "1968-04-02T00:00:00+00:00",
+    "publisher": "Sundhedsdatastyrelsen", 
+    "contact": [
+        {
+            "name": "Mikkel Wermer Steen"
+        }
+    ],
+    "responsible": "Sundhedsdatastyrelsen",
```

Comment:
Uncertain about who is "responsible". For CPR, it was a different department from the publish.

Reply:
I think the name should be removed.

Reply:
Responsible -> https://da.wikipedia.org/wiki/CPR-kontoret